app/Services/Search/Traits/ValidatesTerm.php

```diff
@@ -36,10 +36,10 @@ private function couldBePublicKey(string $term): bool
      */
     private function couldBeUsername(string $term): bool
     {
-        $regex = '/^[a-zA-Z0-9!@$&_.]+$/';
+        $regex = '/^[a-zA-Z0-9!@$&_. ]+$/';
 
         return strlen($term) >= 1
-            && strlen($term) <= 20
+            && strlen($term) <= 30
```

Comment:
@crnkovic FYI: "known wallets" are not related to anything on-chain, they are set by us and can be seen here: https://github.com/ArkEcosystem/common/tree/master/mainnet . They are not related to delegate usernames that are registered on-chain, meaning that they can be anything (hence why they have spaces and why they can surpass the 20 character limit). 

So although this change is fine for now, the 30 character restriction is not something that's actively enforced for those names